code/obj/machinery/manufacturer.dm

```diff
@@ -248,13 +250,12 @@
 					status |= NOPOWER
 					src.build_icon()
 
-	attack_hand(var/mob/user as mob)
-		if (free_resource_amt > 0)
+	attack_hand(mob/user as mob)
```

Comment:
```suggestion
	attack_hand(mob/user)
```

should remove these in non-verb contexts where you see them, doesn't do anything outside of a verb